src/server/protocol.d.ts

```diff
@@ -754,23 +964,47 @@ declare namespace ts.server.protocol {
     export interface CloseRequest extends FileRequest {
     }
 
+    /**
+     * Request to obtain the list of files that should be regenerated if target file is recompiled.

```

Comment:
I would add a note, that this does not generate any outputs. and a reference to `CompileOnSaveEmitFileRequest` for generation.,